packages/slate-editor/src/extensions/placeholders/elements/VideoPlaceholderElement.tsx

```diff
@@ -158,29 +165,45 @@ export function VideoPlaceholderElement({
 
     if (element.provider === PlaceholderNode.Provider.YOUTUBE) {
         return render({
-            action: 'Add',
-            description: 'Insert a YouTube URL and hit enter',
-            placeholder: 'https://youtube.com/video',
-            title: 'YouTube',
+            inputAction: 'Add video',
+            inputDescription: 'Paste a video link and hit Enter',
+            inputPlaceholder: 'https://youtube.com/video',
+            inputTitle: 'YouTube video',
+            title: (props) => <Title {...props} text="Click to insert a YouTube video" />,
+            description: (props) => <Description {...props} text="Add using a video share link" />,
         });
     }
 
     return render();
 }
 
-function Title(props: { isDragOver: boolean; isLoading: boolean }) {
-    if (props.isLoading) {
+interface TitleProps {
+    isDragOver: boolean;
+    isLoading: boolean;
+    text?: string;
+}
```

Comment:
I usually avoid declaring multiple Props interfaces in one file. For the main component there is `Props`. For the rest I do inline definition (as it was before) to not pollute the file visually.

Not like this is important, just sharing my perspective on it :)